cmd/lhsm-plugin-az/mover.go

```diff
@@ -1,301 +1,329 @@
-	action.SetActualLength(total)
-	return nil
-}
-
-// Restore fulfills an HSM Restore request
-func (m *Mover) Restore(action dmplugin.Action) error {
-	util.Log(pipeline.LogDebug, fmt.Sprintf("%s id:%d restore %s %s", m.name, action.ID(), action.PrimaryPath(), action.UUID()))
-	rate.Mark(1)
-
-	var pacer util.Pacer
-
-	start := time.Now()
-	if m.config.Bandwidth != 0 {
-		util.Log(pipeline.LogDebug, fmt.Sprintf("Starting pacer with bandwith %d MBPS\n", m.config.Bandwidth))
-		pacer = util.NewTokenBucketPacer(int64(m.config.Bandwidth*1024*1024), int64(0))
-		defer pacer.Close()
-	}
-	if action.UUID() == "" {
-		return errors.Errorf("Missing file_id on action %d", action.ID())
-	}
-	container, srcObj, err := m.fileIDtoContainerPath(action.UUID())
-	if err != nil {
-		return errors.Wrap(err, "fileIDtoContainerPath failed")
-	}
-
-	contentLen, err := core.Restore(core.RestoreOptions{
-		AccountName:     m.config.AzStorageAccount,
-		BlobEndpointURL: m.config.BlobEndpointURL,
-		ContainerName:   container,
-		ResourceSAS:     m.getSASToken(),
-		BlobName:        srcObj,
-		Credential:      m.cred,
-		DestinationPath: action.WritePath(),
-		Parallelism:     uint16(m.config.NumThreads),
-		BlockSize:       m.config.UploadPartSize,
-		ExportPrefix:    m.config.ExportPrefix,
-		Pacer:           pacer,
-		HTTPClient:      m.httpClient,
-	})
-
-	if err != nil {
-		return errors.Errorf("az.Download() of %s failed: %s", srcObj, err)
-	}
-
-	util.Log(pipeline.LogDebug, fmt.Sprintf("%s id:%d Restored %d bytes in %v from %s to %s", m.name, action.ID(), contentLen,
-		time.Since(start),
-		srcObj,
-		action.PrimaryPath()))
-
-	action.SetActualLength(contentLen)
-	return nil
-}
-
-// Remove fulfills an HSM Remove request
-func (m *Mover) Remove(action dmplugin.Action) error {
-	util.Log(pipeline.LogDebug, fmt.Sprintf("%s id:%d remove %s %s", m.name, action.ID(), action.PrimaryPath(), action.UUID()))
-	rate.Mark(1)
-	if action.UUID() == "" {
-		return errors.New("Missing file_id")
-	}
-
-	container, srcObj, err := m.fileIDtoContainerPath(string(action.UUID()))
-	if err != nil {
-		return errors.Wrap(err, "fileIDtoContainerPath failed")
-	}
-
-	err = core.Remove(core.RemoveOptions{
-		AccountName:     m.config.AzStorageAccount,
-		BlobEndpointURL: m.config.BlobEndpointURL,
-		ContainerName:   container,
-		ResourceSAS:     m.getSASToken(),
-		BlobName:        srcObj,
-		ExportPrefix:    m.config.ExportPrefix,
-		Credential:      m.cred,
-	})
-
-	if err != nil {
-		return errors.Wrap(err, "delete object failed")
-	}
-
-	return nil
-}
+package main
+
+import (
+	"fmt"
+	"math"
+	"net/http"
+	"net/url"
+	"path"
+	"strings"
+	"time"
+
+	core "github.com/wastore/lemur/cmd/lhsm-plugin-az-core"
+	"github.com/wastore/lemur/cmd/util"
+
+	"github.com/pkg/errors"
+
+	"github.com/Azure/azure-pipeline-go/pipeline"
+	"github.com/Azure/azure-storage-blob-go/azblob"
+	"github.com/wastore/go-lustre"
+	"github.com/wastore/go-lustre/fs"
+	"github.com/wastore/go-lustre/status"
+

```

Comment:
what happened here? Github seems to think the entire content changed

Reply:
VSCode did go fmt. Not sure how to fix that.